src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -156,7 +141,7 @@ protected double ParseMetricsInterval(ConfigurationKeyValue kv)
 			return valueInMilliseconds;
 		}
 
-		private bool TryParseTimeInterval(string valueAsString, out double valueInMilliseconds)
+		private static bool TryParseTimeInterval(string valueAsString, out double valueInMilliseconds)
```

Comment:
I see you sometimes adding static (I think mostly because of our `.editorconfig` prefers those) but here you remove it - why? Can't follow and these are so many unrelated changes.